yoga_pilates/data/product_product.xml

```diff
@@ -0,0 +1,134 @@
+    <field name="name">The Flow Membership</field>
+    <field name="image_1920" type="base64" file="yoga_pilates/static/src/binary/product_template/5-image_1920"/>
+    <field name="type">service</field>
+    <field name="service_tracking">partnership</field>
+    <field name="grade_id" ref="res_partner_grade_4"/>
+    <field name="list_price">89.0</field>
+    <field name="service_type">manual</field>
+    <field name="invoice_policy">order</field>
+    <field name="recurring_invoice" eval="True"/>
+    <field name="product_subscription_pricing_ids" eval="[(6,0,[ref('sale_subscription_pricing_5')])]"/>
+    <field name="description_ecommerce"><![CDATA[<p data-oe-version="1.1">Unlimited access to Morning Flow, Yoga Bliss, and Evening Restorative classes.</p>]]></field>
+    <field name="website_sequence">10025</field>
+  </record>
+  <record id="product_product_16" model="product.product">
+    <field name="name">The Tone Membership</field>
+    <field name="image_1920" type="base64" file="yoga_pilates/static/src/binary/product_template/16-image_1920"/>
+    <field name="type">service</field>
+    <field name="service_tracking">partnership</field>
+    <field name="grade_id" ref="res_partner_grade_5"/>
+    <field name="list_price">99.0</field>
+    <field name="service_type">manual</field>
+    <field name="invoice_policy">order</field>
+    <field name="recurring_invoice" eval="True"/>
+    <field name="product_subscription_pricing_ids" eval="[(6,0,[ref('sale_subscription_pricing_6')])]"/>
+    <field name="description_ecommerce"><![CDATA[<p data-oe-version="1.1">Enjoy unlimited Pilates reformer and mat sessions to strengthen, tone, and improve flexibility.</p>]]></field>
+    <field name="website_sequence">10070</field>
+  </record>
+  <record id="event_product.product_product_event" model="product.product" forcecreate="1">
+    <field name="name">Event Registration</field>
+    <field name="image_1920" type="base64" file="yoga_pilates/static/src/binary/product_template/24-image_1920"/>
+    <field name="type">service</field>
+    <field name="service_tracking">event</field>
+    <field name="categ_id" ref="event_product.product_category_events"/>
+    <field name="list_price">30.0</field>
+    <field name="service_type">manual</field>
+    <field name="invoice_policy">order</field>
+    <field name="website_sequence">10085</field>
+    <field name="standard_price">10.0</field>
+  </record>
+  <record id="loyalty.gift_card_product_50_product_template" model="product.template" context="{'create_product_product': False}">
+    <field name="active" eval="False"/>
+  </record>
+  <record id="loyalty.ewallet_product_50_product_template" model="product.template" context="{'create_product_product': False}">
+    <field name="active" eval="False"/>
+  </record>
+  <record id="appointment_account_payment.default_booking_product_product_template" model="product.template" context="{'create_product_product': False}">
```

Comment:
It's weird to archive this product as it's the main point of having appointment_account_payment as a dependency... Could you check with PO?

Reply:
```suggestion
  <record id="appointment_account_payment.default_booking_product" model="product.product" forcecreate="0">
```